collections/iceye_e13b.yaml

```diff
@@ -5,7 +5,7 @@ AdditionalInfoExternal:
     Path: iceye_e13b/README.MD
 Image: iceye_e13b/iceye_e13b.png
 Resolution: Pixel Size = (0.0000060,-0.0000060)
-GeographicalCoverage: -1.245691996, 40.394458816, 8.533407797, 50.049107756
+GeographicalCoverage: 8.516097261, 50.037676694, 8.533407797, 50.049107756
```

Comment:
I am not sure if this change is correct. When checking the collection I see it has data for 3 days (25.08.2020, 12.05.2020, 05.03.2020). The first of these is located at 40.41, -1.26 (lat/long), the 2nd 50.04, 8.52 and the third at the same location as the first. If we understand the `GeographicalCoverage:` as the coordinates that define our bbox (so that all 3 tiles are in it, I think it was correct before).

Reply:
You are right, I had missed the tile at 40.41, -1.26. will revert to the previous one which was correct